tools/stats/utilization_stats_lib.py

```diff
@@ -0,0 +1,58 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from dataclasses_json import DataClassJsonMixin
+
+
+_DATA_MODEL_VERSION = 1.0
+
+
+# data model for test log usage
+@dataclass
+class UtilizationStats:
+    avg: Optional[float] = None
+    max: Optional[float] = None
+
+
+@dataclass
+class UtilizationMetadata(DataClassJsonMixin):
```

Comment:
As this reflects the schema from https://github.com/pytorch/test-infra/pull/6183, I think having a comment here pointing to that schema would be nice in case someone tries to update this in the future to update the schema accordingly